src/generator/event.rs

```diff
@@ -40,6 +41,7 @@ pub enum Event<'a> {
     InlineHtml(Cow<'a, str>),
     Latex(Cow<'a, str>),
     IncludeMarkdown(Box<Events<'a>>),
+    IncludeRustdoc(Box<Rustdoc<'a>>),
```

Comment:
Doing this as normal include is both crazy and amazing! I both love and hate it so much at the same time! :D